### PR TITLE
update UI for android launch

### PR DESCRIPTION
**Pull request by johngruen**
@devinreams  This branch is not done yet, but it's where I'm updating the UI, I'll remove the WIP flag and tag you for review when it's good to go.

### CODE REVIEW

**Comment by devinreams** [comment]
Thanks @johngruen, so far so good 👍 

image

**Comment by johngruen** [comment]
@devinreams i think this is ready for review.... a few notes: 
- i've added `footer_links` in addition to `header_links`. These can serve as site navigation on the new homepage.
- i've added links to the app stores for google and app to the `_config.yml`
- i've left in src files for all weights of Inter even through the site only imports three weights

**Comment by johngruen** [comment]
@devinreams this is ready for real review

File: _layouts/home.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <body class="home">
+    <main class="home__content" aria-label="Content">
+        <div class="home__left">
+          <div class="home__left-inner">
+            <img class="home__title" src="{{relativeBase}}assets/images/lockbox-logo.svg" alt="{{ site.title | escape }}">
+            <h1>{{ content }}</h1>
+          </div>
+        <div class="home__app-buttons">
+          <a class="app-store" href="{{ site.ios_link | escape }}">
+            <img src="{{relativeBase}}assets/images/app-store.svg" alt="Apple App Store" height=" 50px" width="auto">
```
Comment (pdehaan) [inline _layouts/home.html]: I don't think the `height=` tag likes "px" suffixes (that's only a CSS thing IIRC). Probably smart enough to ignore it, but...

Probably same w/ `width="auto"`.

Once this is hosted somewhere, I can run Google Lighthouse and aXe and webhint.io scanners though and check for other potential a11y issues.

**Comment by devinreams** [comment]
Looks great! I just tested all the pages locally and made super sure the FAQ anchors still jump as expected (no reason they wouldn't just saying I _did_ test).

Quick scan of the code, looks fine (@pdehaan has eagle eyes for this though).

@johngruen want us to go ahead and merge so it's live and then we can run more testing and add any more potential todos to #98?

**Comment by devinreams** [comment]
Screen Shot 2019-03-15 at 10 46 25 AM

Screen Shot 2019-03-15 at 10 48 26 AM

**Comment by johngruen** [comment]
Okay @devinreams i fixed @pdehaan 's issue

File: _layouts/home.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          <a class="app-store" href="{{ site.ios_link | escape }}">
+            <img src="{{relativeBase}}assets/images/app-store.svg" alt="Apple App Store" height=" 50">
+          </a>
+          <a class="google-play-store" href="{{ site.android_link | escape }}">
+            <img src="{{relativeBase}}assets/images/gps.svg" height=" 50" alt="Google Play Store">
+          </a>
+        </div>
+        </div>
+        <div class="home__content-right">
+          <img src="./assets/images/devices.png">
```
Comment (pdehaan) [inline _layouts/home.html]: I built this locally and aXe says:

> Images must have alternate text

Firefox_Lockbox___Firefox_Lockbox_project_documentation_and_website

Comment (devinreams) [reply _layouts/home.html]: Added alt text, thanks @pdehaan 🙏

**Pull request merged**